test/E2ETest/TyeRunTests.cs

```diff
@@ -71,6 +71,33 @@ public async Task SingleProjectRunTest()
             });
         }
 
+        [Fact]
+        public async Task SingleProjectRunTestWithDashboard()
```

Comment:
Agreed with your comment, right now this test isn't useful. Let's just delete it for now and verify manually that this works on linux, mac, and windows.